### PR TITLE
[zh]translate Version Skew Policy of release section

### PR DESCRIPTION
**Pull request by howieyuen**
related issue: https://github.com/kubernetes/website/issues/30096

### CODE REVIEW

File: content/zh/releases/version-skew-policy.md [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+Optionally upgrade `kubelet` instances to **{{< skew latestVersion >}}** (or they can be left at **{{< skew prevMinorVersion >}}** or **{{< skew oldestMinorVersion >}}**)
+-->
+### kubelet  {#kubelet-1}
+
+先决条件：
+
+* 与 `kubelet` 通信的 `kube-apiserver` 实例处于 **{{< skew latestVersion >}}** 版本
+
+可选择将 `kubelet` 实例升级到 **{{< skew latestMinorVersion >}}** 版本
+（或者它们可以留在 **{{< skew prevMinorVersion >}}** 或 **{{< skew oldMinorVersion >}}** 版本)
```
Comment (Sea-n) [inline content/zh/releases/version-skew-policy.md:314]: Mixed `（quote)`.
```suggestion
（或者它们可以留在 **{{< skew prevMinorVersion >}}** 或 **{{< skew oldMinorVersion >}}** 版本）
```

**Review by Sea-n — COMMENTED** [review]
Just a little nitpick.

File: content/zh/releases/version-skew-policy.md [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+* `kube-controller-manager`、`kube-scheduler` 和 `cloud-controller-manager`
+  支持 **{{< skew latestVersion >}}** 和 **{{< skew prevMinorVersion >}}** 版本
+
+{{< note >}}
+<!-- 
+If version skew exists between `kube-apiserver` instances in an HA cluster, and these components can communicate with any `kube-apiserver` instance in the cluster (for example, via a load balancer), this narrows the allowed versions of these components.
+-->
+如果 HA 集群中的 `kube-apiserver` 实例之间存在版本偏差，
+并且这些组件可以与集群中的任何 `kube-apiserver`
+实例通信（例如，通过负载均衡器），这会缩小这些组件的允许版本。
```
Comment (tengqm) [inline content/zh/releases/version-skew-policy.md:156]:
```suggestion
实例通信（例如，通过负载均衡器），这会缩小这些组件所允许的版本范围。
```

File: content/zh/releases/version-skew-policy.md [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+<!-- 
+Running a cluster with `kubelet` instances that are persistently two minor versions behind `kube-apiserver` is not recommended:
+
+* they must be upgraded within one minor version of `kube-apiserver` before the control plane can be upgraded
+* it increases the likelihood of running `kubelet` versions older than the three maintained minor releases
+-->
+不建议运行 `kubelet` 实例始终落后 `kube-apiserver` 两个次要版本的集群：
+
+* 它们必须在 `kube-apiserver` 的一个次要版本中升级，然后才能升级控制平面
+* 它增加了运行早于三个维护的次要版本的 `kubelet` 的可能性
```
Comment (tengqm) [inline content/zh/releases/version-skew-policy.md:335]:
```suggestion
* 它增加了运行早于三个处于维护状态的次要版本的 `kubelet` 的可能性
```

**Pull request merged**